www/front_src/src/Resources/SearchHelpTooltip.tsx

```diff
@@ -76,6 +78,13 @@ const Content = ({ onClose }: ContentProps): JSX.Element => {
             {` ${labelSearchByHostNameStartingWith} "FR20" ${labelSearchAndHostAliasContaining} "prod"`}
           </li>
         </ul>
+        <i>
+          <b>{`${labelTips}: `}</b>
+          {`${labelGetRegexHelp} `}
+          <Link href="https://regex101.com" target="_blank">
```

Comment:
There is a lack of security here (about [Attack Surface](https://en.wikipedia.org/wiki/Attack_surface))
You can add `rel="noopener"` or `rel="noreferrer"`
More explanation [here](https://developers.google.com/web/tools/lighthouse/audits/noopener)